core/src/main/java/com/linecorp/armeria/server/ServiceConfig.java

```diff
@@ -230,6 +248,13 @@ public boolean shutdownAccessLogWriterOnStop() {
         return shutdownAccessLogWriterOnStop;
     }
 
+    /**
+     * Returns the {@link Set} of {@link OptOutFeature}s that are disabled for the {@link #service()}.
```

Comment:
Could we also add:

    This method always returns an empty {@link Set} for a non-{@link TransientService},
    because only a {@link TransientService} can opt out from {@link OptOutFeature}s.